Project.toml

```diff
@@ -21,24 +21,24 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 julia = "1.0, 1.1, 1.2, 1.3"
 CDDLib = "0.5.3"
-Distributions = "0.19.2"
+Distributions = "^0.19, ^0.20, ^0.21"
 Documenter = "0.23.0"
-Expokit = "≥ 0.1.0"
-GLPK = "≥ 0.10.0"
-GLPKMathProgInterface = "≥ 0.4.0"
-GR = "≥ 0.39.1"
-IntervalArithmetic = "≥ 0.14.0"
-IntervalMatrices = "≥ 0.2.1"
-JuMP = "0.20.0"
-Makie = "≥ 0.9.4"
-MathProgBase = "≥ 0.7.0"
-Optim = "≥ 0.14.1"
-Plots = "≥ 0.25.1"
-Polyhedra = "0.5.4"
-RecipesBase = "≥ 0.3.0"
-Reexport = "≥ 0.2.0"
-Requires = "≥ 0.4.0"
-TaylorModels = "≥ 0.1.1"
+Expokit = "^0.2"
```

Comment:
Any reason why `Expokit` v0.1 is not allowed anymore? The old version allowed it and it seemed to work.
The same holds for `GR`, `IntervalArithmetic`, `RecipesBase`, `Requires`, and `TaylorModels`.
If there is a reason to upgrade one of those packages, I think we should add a test to capture this necessity.

Reply:
> Any reason why Expokit v0.1 is not allowed anymore? The old version allowed it and it seemed to work.

See: This release establishes compatibility with Julia 0.7/1.0. From now on support for Julia 0.6 will be dropped, https://github.com/acroy/Expokit.jl/releases/tag/v0.2.0

> The same holds for GR, IntervalArithmetic, RecipesBase, Requires, and TaylorModels.
If there is a reason to upgrade one of those packages, I think we should add a test to capture this necessity.

I don't think that we need to add more tests for backwards compatibility  -- if it is not broken, don't fix it :) .

Reply:
Okay, I checked these packages and I agree to upgrade them.